claylib.asd

```diff
@@ -59,9 +59,9 @@
                              (:file "camera-3d")
                              (:file "ray")
                              (:file "ray-collision")
+                             (:file "text")
```

Comment:
`image.lisp` needs `text.lisp` loaded first. I'm not super familiar with ASDF - is there a better way to load `text.lisp` before `image.lisp`? I noticed the line order impacted the load order when I tried to alphabetize this before.

Reply:
Never mind, moving the `image-draw` methods into their respective specializer's files resolves this (b35b6c0a1d9e1ca60681607bd19d1d3ce0590031).

Reply:
Yeah, despite the name, I think they fit better in their own files. Also, please don't alphabetize the ASD file. :-) Those are loaded serially, when we specify `:serial t` like we are. You can also specify dependencies explicitly which is probably more "correct," but messier, IMHO.